src/config_utils.rs

```diff
@@ -22,10 +23,68 @@ use std::path::PathBuf;
 use std::env;
 use rustc_serialize::json;
 use std::io;
+use rustc_serialize::{Decodable, Decoder, Encodable, Encoder};
+
+#[derive(PartialEq, Eq, Hash, Debug, Clone)]
```

Comment:
Looks like you've not pulled the latest if the file your using has this code in it.